README.md
# lang-school-web


changelog.txt
*** Changelog - Ultron Website Template ***

2015.07.01 - version 1.0
	* First release!


ultron/js/map.js
/* ==============================================
MAP -->
=============================================== */

// var locations = [
// 	['<div class="infobox"><h3 class="title"><a href="about.html">OUR USA OFFICE</a></h3><span>NEW YORK CITY 2045 / 65</span><br><span>+90 555 666 77 88</span></div>', -37.801578, 145.060508, 2]
// 	];
// 	var map = new google.maps.Map(document.getElementById('map'), {
// 		zoom: 16,
// 		scrollwheel: false,
// 		navigationControl: true,
// 		mapTypeControl: false,
// 		scaleControl: false,
// 		draggable: true,
// 		styles: [ { "stylers": [ { "hue": "#ffffff" },  {saturation: -100},
// 		{gamma: 2} ] } ],
// 		center: new google.maps.LatLng(-37.801578, 145.060508),
// 		mapTypeId: google.maps.MapTypeId.ROADMAP
// 		});
// 		var infowindow = new google.maps.InfoWindow();
// 		var marker, i;
// 		for (i = 0; i < locations.length; i++) {
// 		marker = new google.maps.Marker({
// 		position: new google.maps.LatLng(locations[i][1], locations[i][2]),
// 		map: map ,
// 		icon: 'images/marker.png'
// 		});
// 		google.maps.event.addListener(marker, 'click', (function(marker, i) {
// 		return function() {
// 		infowindow.setContent(locations[i][0]);
// 		infowindow.open(map, marker);
// 	}
// 	})(marker, i));
// }


ultron/js/custom.js
(function ($) {
  "use strict";

  /* ==============================================
ANIMATION -->
=============================================== */

  new WOW({
    boxClass: "wow", // default
    animateClass: "animated", // default
    offset: 0, // default
    mobile: true, // default
    live: true, // default
  }).init();

  /* ==============================================
    Progress Bar (Skills Bar)
=============================================== */

  $(".progress .progress-bar").progressbar({ transition_delay: 800 });

  /* ==============================================
FUN -->
=============================================== */

  function count($this) {
    var current = parseInt($this.html(), 10);
    current = current + 10;
    $this.html(++current);
    if (current > $this.data("count")) {
      $this.html($this.data("count"));
    } else {
      setTimeout(function () {
        count($this);
      }, 10);
    }
  }
  $(".stat-count").each(function () {
    $(this).data("count", parseInt($(this).html(), 10));
    $(this).html("0");
    count($(this));
  });

  /* ==============================================
BACK TOP -->
=============================================== */

  jQuery(window).scroll(function () {
    if (jQuery(this).scrollTop() > 1) {
      jQuery(".backtotop").css({ bottom: "auto" });
    } else {
      jQuery(".backtotop").css({ bottom: "auto" });
    }
  });
  jQuery(".backtotop").click(function () {
    jQuery("html, body").animate({ scrollTop: "0px" }, 800);
    return false;
  });
  /* ==============================================
MENU HOVER -->
=============================================== */

  $(".dropdown").hover(
    function () {
      $(this).find(".dropdown-menu").stop(true, true).delay(100).fadeIn(300);
    },
    function () {
      $(this).find(".dropdown-menu").stop(true, true).delay(100).fadeOut(300);
    }
  );

  /* ==============================================
SEARCH -->
=============================================== */

  var $ctsearch = $("#dmsearch"),
    $ctsearchinput = $ctsearch.find("input.dmsearch-input"),
    $body = $("html,body"),
    openSearch = function () {
      $ctsearch.data("open", true).addClass("dmsearch-open");
      $ctsearchinput.focus();
      return false;
    },
    closeSearch = function () {
      $ctsearch.data("open", false).removeClass("dmsearch-open");
    };

  $ctsearchinput.on("click", function (e) {
    e.stopPropagation();
    $ctsearch.data("open", true);
  });

  $ctsearch.on("click", function (e) {
    e.stopPropagation();
    if (!$ctsearch.data("open")) {
      openSearch();

      $body.off("click").on("click", function (e) {
        closeSearch();
      });
    } else {
      if ($ctsearchinput.val() === "") {
        closeSearch();
        return false;
      }
    }
  });

  $("#header").affix({
    offset: {
      top: $("#header").height(),
    },
  });

  /* ==============================================
    Tooltip
=============================================== */

  $('[data-toggle="tooltip"]').tooltip();

  $(function () {
    $("#header a[href*=#]:not([href=#])").click(function () {
      if (
        location.pathname.replace(/^\//, "") ==
          this.pathname.replace(/^\//, "") &&
        location.hostname == this.hostname
      ) {
        var target = $(this.hash);
        target = target.length
          ? target
          : $("[name=" + this.hash.slice(1) + "]");
        if (target.length) {
          $("html,body").animate(
            {
              scrollTop: target.offset().top,
            },
            1000
          );
          return false;
        }
      }
    });
  });

  /* ---------------------------------------------
WINDOWS HEIGHT JS -->
 --------------------------------------------- */
  $(".js-height-full").height($(window).height());
  $(".js-height-parent").each(function () {
    $(this).height($(this).parent().first().height());
  });

  /* ==============================================
    Loader Effect
=============================================== */

  $(window).load(function () {
    $("#preloader").delay(500).fadeOut();
    $(".preloader").delay(600).fadeOut("slow");
  });
})(jQuery);

$(".schedule-filter li").on("click", function () {
  var tsfilter = $(this).data("tsfilter");
  $(".schedule-filter li").removeClass("active");
  $(this).addClass("active");
  if (tsfilter == "all") {
    $(".schedule-table").removeClass("filtering");
    $(".ts-item").removeClass("show");
  } else {
    $(".schedule-table").addClass("filtering");
  }
  $(".ts-item").each(function () {
    $(this).removeClass("show");
    if ($(this).data("tsmeta") == tsfilter) {
      $(this).addClass("show");
    }
  });
});

//courses tabs

$(document).ready(function () {
  $("#myTab a").click(function (e) {
    e.preventDefault();
    $(this).tab("show");
  });
});

$('#myTab a[href="#german"]').tab("show");
$('#myTab a[href="#english"]').tab("show");
$('#myTab a[href="#italian"]').tab("show");

$('a[data-toggle="tab"]').on("shown.bs.tab", function (e) {
  e.target; // newly activated tab
  e.relatedTarget; // previous active tab
});


documentation/videoback-doc/js/supersized.3.2.7.bg.js
// JavaScript Document


// supersized
jQuery(function($){
			$.supersized({
				
					// Functionality
					slideshow               :   1,			// Slideshow on/off
					autoplay				:	1,			// Slideshow starts playing automatically
					start_slide             :   1,			// Start slide (0 is random)
					stop_loop				:	1,			// Pauses slideshow on last slide
					random					: 	0,			// Randomize slide order (Ignores start slide)
					slide_interval          :   6000,		// Length between transitions
					transition              :   1, 			// 0-None, 1-Fade, 2-Slide Top, 3-Slide Right, 4-Slide Bottom, 5-Slide Left, 6-Carousel Right, 7-Carousel Left
					transition_speed		:	4000,		// Speed of transition
					new_window				:	0,			// Image links open in new window/tab
					pause_hover             :   0,			// Pause slideshow on hover
					keyboard_nav            :   1,			// Keyboard navigation on/off
					performance				:	2,			// 0-Normal, 1-Hybrid speed/quality, 2-Optimizes image quality, 3-Optimizes transition speed // (Only works for Firefox/IE, not Webkit)
					image_protect			:	1,			// Disables image dragging and right click with Javascript
															   
					// Size || Position						   
					min_width		        :   0,			// Min width allowed (in pixels)
					min_height		        :   0,			// Min height allowed (in pixels)
					vertical_center         :   1,			// Vertically center background
					horizontal_center       :   1,			// Horizontally center background
					fit_always				:	0,			// Image will never exceed browser width or height (Ignores min. dimensions)
					fit_portrait         	:   1,			// Portrait images will not exceed browser height
					fit_landscape			:   0,			// Landscape images will not exceed browser width
															   
					// Components							
					slide_links				:	'false',	// Individual links for each slide (Options: false, 'number', 'name', 'blank')
					thumb_links				:	0,			// Individual thumb links for each slide
					thumbnail_navigation    :   0,			// Thumbnail navigation
					slides 					:  	[			// Slideshow Images
												{image : 'images/background/1.jpg', title : '', thumb : '', url : ''}
												]
					
				});
		    });

documentation/videoback-doc/js/evenings.js
// JavaScript Document


// YTPlayer
    $(function(){
      $(".player").mb_YTPlayer();
    });
	
	
// screen loader
            $(window).load(function() {
                // $('body').css('overflow', 'hidden');
                $('.screen-loader').fadeOut('slow');
            });


// preload
$(document).ready(function() {
	$('#preload').css({display: 'block'});
});


// preload function
$(window).load(preLoader);
function preLoader() {
	setTimeout(function() {

		$('#preload').delay(250).fadeOut(1500);
		$('.intro-wrapper').delay(2000).css({display: 'none'}).fadeIn(3000);

	});
};


// fire home
	$("#fire-home").click(function(e) {
		e.preventDefault();
		$(".current").fadeOut("slow", function() {
			$(".upper-page").fadeIn("slow");      
			$(".current").removeClass("current");
			$(".upper-page").addClass("current");
		});
	});
	
	
// fire about
	$("#fire-about").click(function(e) {
		e.preventDefault();
		$(".current").fadeOut("slow", function() {
			$("#about").fadeIn("slow");
			$(".current").removeClass("current");
			$("#about").addClass("current");
		});
	});
	
	
// fire contact
	$("#fire-contact").click(function(e) {
		e.preventDefault();
		$(".current").fadeOut("slow", function() {
			$("#contact").fadeIn("slow");
			$(".current").removeClass("current");
			$("#contact").addClass("current");
		});
	});
	
	
// fire home mobile
	$("#fire-home-mobile").click(function(e) {
		e.preventDefault();
		$(".current").fadeOut("slow", function() {
			$(".upper-page").fadeIn("slow");      
			$(".current").removeClass("current");
			$(".upper-page").addClass("current");
		});
	});
	
	
// fire about mobile
	$("#fire-about-mobile").click(function(e) {
		e.preventDefault();
		$(".current").fadeOut("slow", function() {
			$("#about").fadeIn("slow");
			$(".current").removeClass("current");
			$("#about").addClass("current");
		});
	});
	
	
// fire contact mobile
	$("#fire-contact-mobile").click(function(e) {
		e.preventDefault();
		$(".current").fadeOut("slow", function() {
			$("#contact").fadeIn("slow");
			$(".current").removeClass("current");
			$("#contact").addClass("current");
		});
	});
	

// fire closer
	$("#fire-about-closer, #fire-contact-closer").click(function(e) {
		e.preventDefault();
		$(".current").fadeOut("slow", function() {
			$(".upper-page").fadeIn("slow");      
			$(".current").removeClass("current");
			$(".upper-page").addClass("current");
		});
	});
	
	
// menu active state
$('a.menu-state').click(function() {
    $('a.menu-state').removeClass("active");
    $(this).addClass("active");
});
	

// niceScroll
$(document).ready(function () {
    $("body").niceScroll({
       cursorborder: "1px solid #fff",
	   cursorborderradius: "0px",
	   zindex: "9999",
       cursorcolor: "#fff",
       background: "rgba(255, 255, 255, 0.1)",
       scrollspeed: "60",
       mousescrollstep: "40"
    });
});


// niceScroll || scrollbars resize
$("body").getNiceScroll().resize();


// newsletter animation
$(function(){
		   
			$("ul.button").delay(2200).animate({
			   right: 0
			}, 800);
			
			$(".newsletter-animation .newsletter-bg").delay(1400).animate({
			   height: 69,
			   top: 0
			}, 800);
			
			$("a.close").click(function(){
			$(".newsletter-animation .newsletter-bg").stop().animate({
			   height: 0,
			   top: 85
			}, 300);
				
			$("ul.button a.open").stop().animate({
			   left: 0
			}, 300);
				
			$("ul.button a.close").stop().animate({
			   left: -30
			}, 300);
			});
			
			
			$("ul.button a.open").click(function(){
			$(".newsletter-animation .newsletter-bg").stop().animate({
			   height: 69,
			   top: 0
			}, 800);
				
			$("ul.button a.open").stop().animate({
			   left: 30
			}, 300);
				
			$("ul.button a.close").stop().animate({
			   left: 0
			}, 300);
			});

});


// function for the vertical scrolling effect
$(function() {
    $('.go-under, .go-way-under, .go-way-up').bind('click',function(event){
        var $anchor = $(this);
 
        $('html, body').stop().animate({
            scrollTop: $($anchor.attr('href')).offset().top
        }, 1000,'easeInOutExpo');
        /*
        if you don't want to use the easing effects:
        $('html, body').stop().animate({
           scrollTop: $($anchor.attr('href')).offset().top
        }, 1000);
        */
        event.preventDefault();
    });
});


// twitter ticker
      jQuery(function($){
        $("#ticker").tweet({
          username: "enihilo",
          page: 1,
          avatar_size: 0,
          count: 20,
          loading_text: ""
        }).bind("loaded", function() {
          var ul = $(this).find(".tweet_list");
          var ticker = function() {
            setTimeout(function() {
              ul.find('li:first').animate( {marginTop: '-75px'}, 500, function() {
                $(this).detach().appendTo(ul).removeAttr('style');
              });
              ticker();
            }, 8000);
          };
          ticker();
        });
      });
	  
	  
// twitter ticker settings
$(document).ready(function(){
        $(this).find(".tweet_list").list_ticker({
                speed: 8000,
                effect: 'fade' // fade, slide
        })             
})


// mobile-detect
	var isMobile = {
        Android: function() {
            return navigator.userAgent.match(/Android/i);
        },
        BlackBerry: function() {
            return navigator.userAgent.match(/BlackBerry/i);
        },
        iOS: function() {
            return navigator.userAgent.match(/iPhone|iPad|iPod/i);
        },
        Opera: function() {
            return navigator.userAgent.match(/Opera Mini/i);
        },
        Windows: function() {
            return navigator.userAgent.match(/IEMobile/i);
        },
        any: function() {
            return (isMobile.Android() || isMobile.BlackBerry() || isMobile.iOS() || isMobile.Opera() || isMobile.Windows());
        }
    };